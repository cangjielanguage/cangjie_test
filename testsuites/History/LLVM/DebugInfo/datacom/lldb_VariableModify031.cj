// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_VariableModify031_expected.txt ../basecase/024_debug_variable.cj
// EXEC: cjc -g 024_debug_variable.cj
// EXEC: python3 DatacomTest.py lldb_VariableModify031_expected.txt



