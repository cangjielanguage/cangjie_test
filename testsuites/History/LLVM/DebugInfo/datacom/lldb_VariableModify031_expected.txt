lldb,lldb 024_debug_variable,024_debug_variable
b,b 35,024_debug_variable.cj:35
r,r,-> 35
cj_set,cj_set var_uint64 = "aaa",error
n,n,-> 39
cj_plocals,cj_plocals,var_uint64 = 1
cj_plocals,cj_plocals,f_uint64 = 9